Apps/UnitTests/Shared/Tests.cpp

```diff
@@ -28,11 +30,9 @@ void SetExitCode(const Napi::CallbackInfo& info)
     exitCode.set_value(info[0].As<Napi::Number>().Int32Value());
 }
 
-int run()
+int run(std::unique_ptr<Babylon::Graphics> graphicsObject)
```

Comment:
nit: didn't notice the first time, but it probably should be this:
```suggestion
int Run(std::unique_ptr<Babylon::Graphics> graphicsObject)
```